strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/PypiArtifactGenerator.java

```diff
@@ -100,34 +105,39 @@ public Path generateArtifact(URI uri,
             throws IOException
     {
         PypiArtifactCoordinates coordinates = PypiArtifactCoordinates.parse(FilenameUtils.getName(uri.toString()));
-        return generateArtifact(coordinates);
+        return generateArtifact(coordinates, size);
     }
 
-    public Path generateArtifact(PypiArtifactCoordinates coordinates)
+    public Path generateArtifact(PypiArtifactCoordinates coordinates, long size)
```

Comment:
Could you please rename `size` to `byteSize` here and across the pull request? Thanks! :)

@wpater has already been following this naming convention.